ecs_web_service_template/variables.tf

```diff
@@ -127,6 +127,10 @@ variable "container_definitions" {
   description = "aws_ecs_task_definition.container_definitions, 'containerDefinitions' of the ECS task JSON"
 }
 
+variable "ecs_deployment_maximum_percent" {
+  description = "aws_ecs_service.ecs_deployment_maximum_percent (0 to 200)"
```

Comment:
terraform (>= 0.12) supports `null`. 
Could you set it as default?

https://www.terraform.io/docs/configuration/expressions.html
> If you set an argument of a resource or module to null, Terraform behaves as though you had completely omitted it — it will use the argument's default value if it has one, or raise an error if the argument is mandatory.

Reply:
I fixed it and confirmed it.